### PR TITLE
Add instruction for running unit tests in parallel

### PR DESCRIPTION
**Pull request by ashfame**
This PR updates the contributing guide with the instruction for running unit tests in parallel.

### CODE REVIEW

**Review by babolivier — CHANGES_REQUESTED** [review]
Looks good to me, but can you add a changelog file please?

**Comment by ashfame** [comment]
@babolivier Done!

**Review by babolivier — APPROVED** [review]
Thanks! :slightly_smiling_face:

File: changelog.d/13928.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Added instruction to contributing guide for running unit tests in parallel. Contributed by @ashfame.
```
Comment (babolivier) [inline changelog.d/13928.doc:1]: Quick fixup for consistency

```suggestion
Add instruction to contributing guide for running unit tests in parallel. Contributed by @ashfame.
```

**Pull request merged**